build.gradle

```diff
@@ -18,6 +18,7 @@ plugins {
 }
 
 repositories {
+    jcenter()
```

Comment:
this can probably be removed (presumably it came from the klaxon dependency?)

Reply:
fort that, yes